project/Publish.scala

```diff
@@ -0,0 +1,42 @@
+import java.io.File
+
+import sbt.Keys._
+import sbt._
+
+object Publish {
+
+  def skipSettings: Seq[Def.Setting[_]] =
+    Seq(
+      publish / skip := true
+    )
+
+  def settings: Seq[Def.Setting[_]] =
+    Seq(
+      resolvers ++= Seq(
```

Comment:
? It uses dependencies from there ? For me only publish to through setting thereafter